.github/workflows/CI.yml

```diff
@@ -51,21 +55,23 @@ jobs:
           echo "matrix.os:         ${{ matrix.os }}"
           export INSTALLDIR="${HOME}/${INSTALLDIR}"
           echo "::set-output name=install_dir::${INSTALLDIR}"
-          echo "::set-output name=home_dir::${HOME}"
           echo "::set-env name=INSTALLDIR::${INSTALLDIR}"
+          echo "::set-output name=home_dir::${HOME}"
+          echo "::set-output name=taucmdr_system::${INSTALLDIR}/${__TAUCMDR_SYSTEM_PREFIX__}"
+          echo "::set-env name=__TAUCMDR_SYSTEM_PREFIX__::${INSTALLDIR}/${__TAUCMDR_SYSTEM_PREFIX__}"
 
       - name: Cache dependencies
         uses: actions/cache@v1
         with:
           path: ${{ steps.info.outputs.install_dir}}/conda
-          key: ${{ runner.os }}-pip${{ matrix.python_version }}-${{ hashFiles('**/requirements-dev.txt') }}
+          key: ${{ runner.os }}-pip${{ matrix.python_version }}-${{ hashFiles('**/requirements-dev.txt') }}-${{ hashFiles('**/CI.yml') }}-${{ github.sha }}
```

Comment:
The python/pip cache should be invalidated when CI.yml changes otherwise paths in, e.g., shebangs, may be incorrect.